linear/linear.go

```diff
@@ -7,13 +7,19 @@ func main() {
 	fmt.Println("Enter A and B:")
 	var A, B float64
 	fmt.Scan(&A, &B)
-	if A != 0 {
+	if A > 0 {
 		x := -B / A
 		fmt.Printf("x > %v", x)
-	} else {
-		if B != 0 {
+	}
+	if A < 0 {
```

Comment:
Please use `else` statements. Instead of
```go
if condition1 { ... }
if condition2 { ... }
if condition3 { ... }
```
it should be
```go
if condition1 {
  ...
} else if condition2 {
  ...
} else { // <- Notice how you don't even have to test the value here.
  ...
}
```